save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/DeleteOrganizationButton.kt

```diff
@@ -0,0 +1,117 @@
+/**
+ * Utilities for cli args parsing
+ */
+
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
+package com.saveourtool.save.frontend.components.views.usersettings
+
+import com.saveourtool.save.frontend.components.modal.displayModal
+import com.saveourtool.save.frontend.utils.*
+import com.saveourtool.save.frontend.utils.noopLoadingHandler
+import csstype.ClassName
+import react.ChildrenBuilder
+import react.FC
+import react.Props
+import react.dom.html.ReactHTML.button
+import react.dom.html.ReactHTML.div
+import react.useState
+
+
+/**
+ * Button for delete organization
+ *
+ * @return noting
+ */
+val deleteOrganizationButton: FC<DeleteOrganizationButtonProps> = FC { props ->
+    val displayButtonEntry = {}
+    val windowOpenness = useWindowOpenness()
+    val (displayTitle, putDisplayTitle) = useState<String?>(null)
```

Comment:
```suggestion
    val (displayTitle, putDisplayTitle) = useState<String>()
```

Reply:
Empty parentheses mean `undefined` don't they?
Probably it is the same, but `useState<String?>(null)` seems more clear

Reply:
Done